File: joseffe-barroso/Java/src/entrada_processamento_saida/aula04/For/Exercise37.java
package For;

import java.util.Scanner;

public class Exercise37 {

	public static void main(String[] args) {
		Scanner scanner = new Scanner(System.in);

		String confirmation;
		int value = 1, result, index;
		
		for (index = 1; value < 20; index++) {
			for (index = 1; index < 21; index++) {
				 result = value * index;
				 System.out.printf("\n%d X %d = %d", value, index, result);
				 
				 if (index == 20) {
					 System.out.print("\nDigite 'OK' para continuar: ");
					 confirmation = scanner.next();
					 
					 while (!confirmation.equals("OK")) {
						 System.out.print("Digite 'OK' para continuar: ");
						 confirmation = scanner.next();
					 } 
					 
					 value = value + 1;	 
				 }
			}
		}
		System.out.print("Thank you");
	}

}

File: joseffe-barroso/Python/aula04/for/exercise37.py
value = 1

for index in range(1, 20, 1):
    for index in range(1, 21, 1):
        result = value * index 
        print(f"{value} X {index} = {result}")
        index = index + 1

        
        if (index == 21):
            confirmation = input("Digite 'OK' para continuar: ")
            
            if (confirmation == 'OK'):
                value = value + 1
            else:
                confirmation = input("Digite 'OK' para continuar: ")

print("Thank you")